index.bs

```diff
@@ -6507,6 +6510,78 @@ when passed to a [=platform object=] expecting that type, and how IDL values
 of that type are <dfn id="dfn-convert-idl-to-ecmascript-value" export lt="converted to an ECMAScript value|converted to ECMAScript values">converted to ECMAScript values</dfn>
 when returned from a platform object.
 
+<div algorithm>
```

Comment:
IMO it would be reasonable to create a subsection for integer types, and give each type a heading under it but also give these supporting abstract ops a heading under it. (Abstract ops after the integer types, I think.)